playbooks/provision_vm/manage_template.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Playbook to provision a Template on VMware
```

Comment:
This wouldn't create a new template but it would create a VM using an existing template

Reply:
I thought it one more time - you're right
you should use - is_template https://docs.ansible.com/ansible/latest/collections/community/vmware/vmware_guest_module.html#parameter-is_template 
cc @mikemorency